client/src/views/WorkspaceDetail.vue

```diff
@@ -80,19 +80,21 @@
             text
           >
             <item-panel
-              ref="tablePanel"
-              title="Tables"
-              :items="tables"
+              ref="graphPanel"
+              title="Networks"
+              :items="graphs"
```

Comment:
Why did you change the order of these? Better UX?

Reply:
Alex requested this change to make "networks" front-and-center. (That's what I meant by "make network list primary" in the issue title, but that was probably unclear.)

Reply:
Gotcha!